training/cli.py

```diff
@@ -7,19 +7,26 @@
 import requests
 
 
+class AuthentificationError(Exception):
+    def __init__(self, message=None, errors=None):
+        if errors and "UnrecognizedClientException" in str(errors):
+            message = "AWS authentification incomplete. Make sure all credentials are set correctly including `export AWS_PROFILE=<profile-name>`"
```

Comment:
change "authentification" to "authentication" (minor misspelling) in the `message` value